packages/in_app_purchase/ios/Tests/InAppPurchasePluginTest.m

```diff
@@ -204,4 +205,39 @@ - (void)testRefreshReceiptRequest {
   XCTAssertTrue(result);
 }
 
+- (void)testGetPendingTransactions {
+  XCTestExpectation* expectation = [self expectationWithDescription:@"expect success"];
+  FlutterMethodCall* call =
+      [FlutterMethodCall methodCallWithMethodName:@"-[SKPaymentQueue transactions]" arguments:nil];
+  SKPaymentQueue* mockQueue = OCMClassMock(SKPaymentQueue.class);
+  NSDictionary* transactionMap = @{
+    @"transactionIdentifier" : @"567",
+    @"transactionState" : @(SKPaymentTransactionStatePurchasing),
```

Comment:
A SKPaymentTransaction's `transactionIdentifier` is nil when its state is `SKPaymentTransactionStatePurchasing `.

I guess we should either change above line to:
 `@"transactionIdentifier" : [NSNull null],` 
or change this line to :
`@"transactionState" : @(SKPaymentTransactionStatePurchased),`

Reply:
done, updated to `@"transactionIdentifier" : [NSNull null],`